src/main/java/org/cactoos/iterable/Matched.java

```diff
@@ -66,12 +69,13 @@ public Matched(final Iterable<X> fst, final Iterable<X> snd) {
      */
     public Matched(
         final BiFunc<X, X, Boolean> fnc,
```

Comment:
@andreoss here I think it should be `BiFunc<? super X, ? super X, Boolean>`

Reply:
@victornoel Fixed